app/views/layouts/_navigation.erb

```diff
@@ -0,0 +1,66 @@
+<div class="navbar navbar-default navbar-static-top">
+  <div class="container">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-responsive-collapse">
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+    </button>
+    <a class="navbar-brand" href="/">TrainHard</a>
+    <div class="navbar-collapse collapse navbar-responsive-collapse">
+      <ul class="nav navbar-nav col-md-6">
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Disciplines <span class="caret"></span></a>
+          <ul class="dropdown-menu">
+            <li><%= link_to "Disciplines", disciplines_path %></li>
+            <% if current_user&.admin? %>
```

Comment:
we have a nullobject for that, please remove `&.`

Reply:
thanks mate!